pkg/services/serviceaccounts/manager/roles.go

```diff
@@ -29,4 +51,51 @@ var (
 		},
 		Grants: []string{"Admin"},
 	}
-)
+
+	if err := ac.DeclareFixedRoles(role); err != nil {
+		return err
+	}
+
+	apikeyAdminReadRole := accesscontrol.RoleRegistration{
+		Role: accesscontrol.RoleDTO{
+			Version:     1,
+			Name:        apikeyAdminRead,
+			Description: "Gives access to list apikeys.",
```

Comment:
We want to add a display name and group to each role. Also lets loop in @osg-grafana for the names / descriptions